packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5063,17 +5092,19 @@ class _ScribblePlaceholder extends WidgetSpan {
   final Size size;
 
   @override
-  void build(ui.ParagraphBuilder builder, { double textScaleFactor = 1.0, List<PlaceholderDimensions>? dimensions }) {
+  void build(ui.ParagraphBuilder builder, {
+    TextScaler textScaler = TextScaler.noScaling,
+    List<PlaceholderDimensions>? dimensions,
+  }) {
     assert(debugAssertIsValid());
     final bool hasStyle = style != null;
     if (hasStyle) {
-      builder.pushStyle(style!.getTextStyle(textScaleFactor: textScaleFactor));
+      builder.pushStyle(style!.getTextStyle(textScaler: textScaler));
     }
     builder.addPlaceholder(
-      size.width,
-      size.height,
+      size.width * textScaler.textScaleFactor,
+      size.height * textScaler.textScaleFactor,
```

Comment:
Another place that should use `textScaler.scale` maybe?

Reply:
How are you keeping track of what changes in this PR are final and which ones needs to be revisited in a follow-up so we don't forget about them? Would it make sense to just keep using the deprecated API in the places that need follow-up changes to make it super clear that they need work? Alternatively, we could add TODOs everywhere, but given how wide-spread this is, I am a little wary of that...

Reply:
I can find references to `TextScaler.textScaleFactor`. Pretty much all of them will have to be removed.